quarterly_reporting.py

```diff
@@ -0,0 +1,271 @@
+import numpy as np
+import pandas as pd
+from pypgrest import Postgrest
+from sodapy import Socrata
+
+import os
+
+POSTGREST_ENDPOINT = os.getenv("POSTGREST_ENDPOINT")
+POSTGREST_TOKEN = os.getenv("POSTGREST_TOKEN")
+SO_WEB = os.getenv("SO_WEB")
+SO_TOKEN = os.getenv("SO_TOKEN")
+SO_KEY = os.getenv("SO_KEY")
+SO_SECRET = os.getenv("SO_SECRET")
+
+# Socrata dataset IDs
+SUBPROJECT_DATASET = os.getenv("SUBPROJECT_DATASET")
+FDU_DATASET = os.getenv("FDU_DATASET")
+EXPENSES_DATASET = os.getenv("EXPENSES_DATASET")
+
+SUBPROJECT_TABLES = [
+    "fdus_2020_bond",
+    "subprojects_with_appropriations",
+    "subprojects_with_budget",
+]
+EXPENSES = "fdu_expenses_quarterly"
+SPEND_PLAN = "quarterly_spend_plan"
+FDU_METADATA = "fdu_metadata_quarterly"
+
+
+
+FY_MONTHS = {
+    "OCT": "00",
+    "NOV": "01",
+    "DEC": "02",
+    "JAN": "03",
+    "FEB": "04",
+    "MAR": "05",
+    "APR": "06",
+    "MAY": "07",
+    "JUN": "08",
+    "JUL": "09",
+    "AUG": "10",
+    "SEP": "11",
+    "P13": "11",  # Putting period 13 in September
```

Comment:
expenses for the current FY that occur in the next FY are put in this "period 13"

Reply:
and this wont double count the expenses? I guess this more a question for Meredith....

Reply:
It shouldn't. It'll just group those expenses into September

Reply:
so our accountants use a 13 month calendar eh? ☠️